openhtf/util/units/bin/units_from_xls.py

```diff
@@ -95,21 +99,23 @@ def code_generator(sheet):
     
     # Find the indices for the columns we care about.
     for idx, cell in enumerate(rows.next()):
-      if cell.value in COLUMN_NAMES:
+      if cell.value in column_names:
         col_indices[cell.value] = idx
 
     # loop over all remaining rows and pull out units.
     for row in rows:
-      name = row[col_indices[COLUMN_NAMES[0]]].value
-      code = row[col_indices[COLUMN_NAMES[1]]].value
-      suffix = multi_replace(row[col_indices[COLUMN_NAMES[2]]].value,
-                             {"'": "\\\'", '"': '\\\"'})
+      name = row[col_indices[column_names[0]]].value
+      code = row[col_indices[column_names[1]]].value
+      suffix = row[col_indices[column_names[2]]].value.replace("'", "\\\'")
       if name in seen:
         continue
       seen.add(name)
 
-      yield 'UNITS[\'%s\'] = Unit(\'%s\', \'%s\', \'%s\')\n' % (
-          multi_replace(name.upper(), REPLACEMENTS), name, code, suffix)
+      yield "UNITS['%s'] = Unit('%s', '''%s''')\n" % (
+          multi_replace(name.upper(),
+                        UNIT_KEY_REPLACEMENTS),
```

Comment:
this indentation is super confusing, looks like all 4 lines are args to multi_replace.  i'd put `UNIT_KEY_REPLACEMENTS),` on the previous line